Xamarin.Forms.Platform.Android/VisualElementRenderer.cs

```diff
@@ -135,6 +134,40 @@ public void UpdateLayout()
 			Performance.Stop(reference);
 		}
 
+		protected int TabIndex { get; set; } = 0;
+
+		protected bool TabStop { get; set; } = true;
+
+		protected void UpdateTabStop() => TabStop = Element.IsTabStop;
+
+		protected void UpdateTabIndex() => TabIndex = Element.TabIndex;
+
+		public override AView FocusSearch(AView focused, [GeneratedEnum] FocusSearchDirection direction)
+		{
+			VisualElement element = Element as VisualElement;
+			int maxAttempts = 0;
+			var tabIndexes = element?.GetTabIndexesOnParentPage(out maxAttempts);
+			if (tabIndexes == null)
+				return null;
+
+			int tabIndex = element.TabIndex;
+			AView control = null;
+			int attempt = 0;
+			bool forwardDirection = !(
+				(direction & FocusSearchDirection.Backward) != 0 ||
```

Comment:
why aren't `FocusSearchDirection.Right` and `FocusSearchDirection.Down` dealt with here? Did you mean it?